### PR TITLE
blog: December newsletter

### PR DESCRIPTION
**Pull request by orrabin**

### CODE REVIEW

File: _posts/2017-12-28-foreman-community-newsletter---december-2017.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+For those who missed it, the past month there were discussions and
+demos showing Discourse. After a migration poll last month the move to
+Discourse got a lot of support.
+The results were discussed [here](https://youtu.be/g8XnAKfG9xY?t=99). 
+
+### Katello 3.5 released
+
+On the 7th of Dec Katello 3.5 was released RC2, see the [post][k_3_5] for details.
+The release fixed 28 bugs found in RC2 so thank you to all that tested,
```
Comment (mmoll) [inline _posts/2017-12-28-foreman-community-newsletter---december-2017.md]: this reads a bit unclear... The two message are
1. Katello 3.5 (final) is released
2. 28 issues were fixed from RC2 to the final release

correct?

Comment (orrabin) [reply _posts/2017-12-28-foreman-community-newsletter---december-2017.md]: copy+paste error :) yes both are correct, I'll fix it

**Review by GregSutcliffe — APPROVED** [review]

File: _posts/2017-12-28-foreman-community-newsletter---december-2017.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+excerpt: |
+  Migrating from mailing lists, Katello 3.5, and the usual round-up
+---
+
+Happy holidays!
+
+### Migrating from Google Groups
+
+Our mailing lists are moving to Discourse on 1st Jan!
+See the [discourse_announcement] for more details.
```
Comment (mmoll) [inline _posts/2017-12-28-foreman-community-newsletter---december-2017.md]: text for the link missing?

**Comment by orrabin** [comment]
@mmoll I fixed the missing test, thanks.

File: _posts/2017-12-28-foreman-community-newsletter---december-2017.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+* Community Demo - 4th Jan  (https://www.youtube.com/watch?v=BVTiwSEMcyw)
+
+Time again for the latest round-up of action from the community, covering core,
+proxies, plugins, and more.  As always, you're welcome to participate in the
+show, either via YouTube Live chat, or via our IRC channel
+(Freenode/#theforeman).
+
+#### Upcoming Conferences
+
+* DevConf.cz: Look out for the Foreman booth at [DevConf](https://devconf.cz/) in Brno 26-28 Jan
```
Comment (tbrisker) [inline _posts/2017-12-28-foreman-community-newsletter---december-2017.md]: For consistency - make the link from the conference name in the beginning of the row

File: _posts/2017-12-28-foreman-community-newsletter---december-2017.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+### Katello 3.5 released
+
+On the 7th of Dec Katello 3.5 was released, see the [post][k_3_5] for details.
+The release fixed 28 bugs found in RC2 so thank you to all that tested,
+commented and fixed! As always, we'd love your feedback, so do report
+any issues you find to the [Katello bug tracker][k_issues]!
+
+### Upcoming Events
+
+* Community Demo - 4th Jan  (https://www.youtube.com/watch?v=BVTiwSEMcyw)
```
Comment (tbrisker) [inline _posts/2017-12-28-foreman-community-newsletter---december-2017.md:35]: Shouldn't this also be under the Community Demos header below? Also maybe make the link from the text?

File: _posts/2017-12-28-foreman-community-newsletter---december-2017.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+Watch for Discourse updates, fact importing improvements and significant
+performance improvements in Content View publishing
+
+
+* [UX review - breadcrumbs and item switcher](https://www.youtube.com/watch?v=4fX_kjPlzVw)
+The change to vertical navigation made two-pane a little less comfortable because
+the space to edit became smaller. Additionally, some users reported confusion
+around navigation context as well as inconsistencies in regards to two-pane navigation. 
+This review will focus on the addition of breadcrumbs that would improve navigation
+compared to two pane. We'd be glad to get feedback on this proposed change.
```
Comment (tbrisker) [inline _posts/2017-12-28-foreman-community-newsletter---december-2017.md]: This already happened, but the text is in future tense

**Review by tbrisker — COMMENTED** [review]
just a few inline comments

File: _posts/2017-12-28-foreman-community-newsletter---december-2017.md (hunk 4 above)
Comment (orrabin) [reply _posts/2017-12-28-foreman-community-newsletter---december-2017.md:35]: No, it's always like this. The next community demo is under upcoming events. If there were any other deep dives coming up they would also be here.

**Comment by mmoll** [comment]
merged, thanks @orrabin!

**Pull request merged**